### DISCUSSION TITLE
[功能请求] 效仿microsoft powertoys, 对顶置程序窗口添加边框

### DISCUSSION THREAD

**Discussion by xmbai9507** [shuax/MouseInc.Settings#287] [Ideas]
目前程序顶置欠缺状态提示, 让人不知道当前窗口是否已经顶置, 尤其是在离开电脑一段时候之后再回来. 如果能效仿microsoft powertoys添加提示边框, 那就太好了. 比如下图窗口的蓝框.
![image](https://user-images.githubusercontent.com/65625632/173174438-1bb066db-df93-46aa-a24f-832aa8eef9cf.png)

**Comment by kevzcr** [comment]
这绝对是个好建议。

**Comment by shuax** [comment]
可以一起用

**Reply by xmbai9507** [reply]
一起用倒也不是不行, 但就为这个提示边框再开一个powertoys有点犯不上, 还是希望如有闲暇能改进一下内置的窗口顶置功能, 感谢.

**Reply by 6Tom** [reply]
窗口置顶只是简单调用了windows自己的接口，就是发送一条很简单的命令，你提这些都需要单独开发出比较复杂的一套逻辑，你可以看看powertoys中，光always on top这一个模块就有1.2MB，比整个mouseinc都大。

说句题外的，我就单纯好奇，无意冒犯，powertoys都不常驻，你用powertoys都是做些什么事情？唤醒、FancyZones、鼠标实用工具、PowerToys Run、视频会议静音，所有这些模块无一例外，全部需要powertoys常驻才能使用啊，一个也不用的话为啥要装这软件呢

**Reply by xmbai9507** [reply]
> 窗口置顶只是简单调用了windows自己的接口，就是发送一条很简单的命令，你提这些都需要单独开发出比较复杂的一套逻辑，你可以看看powertoys中，光always on top这一个模块就有1.2MB，比整个mouseinc都大。

那也可以尝试调用windows的阴影来充当提示边框嘛, 我就给作者提供一下思路, 变通方法很多.

> 说句题外的，我就单纯好奇，无意冒犯，powertoys都不常驻，你用powertoys都是做些什么事情？唤醒、FancyZones、鼠标实用工具、PowerToys Run、视频会议静音，所有这些模块无一例外，全部需要powertoys常驻才能使用啊，一个也不用的话为啥要装这软件呢

所以我不用powertoys啊, 多数功能纯鸡肋, 有其它软件提供更好的使用体验. 有特异价值的也就fancyzones和always on top, 但前者win11自带的就够我用了, 哪怕真要用那也是现用现开, 后者用mouseinc手势操作替代.
